tests/args.py

```diff
@@ -164,6 +169,42 @@ def test_commands_together(self):
         self.assertEqual(args.commands, [["test", "args"], ["restart", "all"]])
 
 
+class TestProfileArguments(unittest.TestCase):
+
```

Comment:
It is ultimately testing the functionality of the standard library.  It is a slippery slope to begin testing that the built in `ConfigParser` module behaves as advertised in automated tests.  If we feel like it is worthwhile to write automated tests to continually confirm this behavior, I can put something together.